### ISSUE TITLE
Clarify the points badge of unofficial submissions

### ISSUE THREAD

**Issue by markkuriekkinen** [apluslms/a-plus#994]
The points badge of unofficial submissions should have clearer labels/tooltips: give detailed and explicit information to the user so that it is easy to understand. For example, the tooltip should say if the deadline was exceeded or the max submissions limit was exceeded instead of listing all alternatives.

![image](https://user-images.githubusercontent.com/6130378/155324079-8c30c966-441e-4bc0-b340-63dc8c03281c.png)
